infrastructure/code/template/details.html

```diff
@@ -385,10 +356,53 @@
       border-color: var(--vscode-button-hoverBackground);
       color: var(--vscode-button-foreground);
     }
+
+    .ignore-details-tab,
+    .fix-analysis-tab,
+    .vuln-overview-tab {
+      text-transform: uppercase;
+    }
+
+    .ignore-details {
+      display: flex;
+      flex-direction: column;
+      align-items: flex-start;
+      padding: 20px 0;
+      gap: 12px;
+      box-sizing: border-box;
+      width: 100%;
+    }
+
+    .ignore-details-row {
+      display: flex;
+      justify-content: space-between;
+      width: 100%;
+      align-items: flex-start;
+    }
+
+    .ignore-details-row {
+      display: flex;
+      justify-content: space-between;
+      width: 100%;
+      align-items: flex-start;
+    }
+
+    .ignore-details-label {
+      width: 30%;
+      text-align: left;
+      font-weight: bold;
+      padding-right: 10px;
+    }
+
+    .ignore-details-value {
+      flex-grow: 1;
+      text-align: left;
+      align-self: stretch;
+    }
   </style>
 </head>
 
-<body>
+<body class="snyk-ide">
```

Comment:
Question: What's this here for?

Reply:
It a leftover from my tests with scoped CSS, we can delete it 💪